packages/fx-core/src/core/question.ts

```diff
@@ -78,7 +80,7 @@ export function createAppNameQuestion(validateProjectPathExistence = true): Text
         const validateResult = jsonschema.validate(appName, schema);
         if (validateResult.errors && validateResult.errors.length > 0) {
           if (validateResult.errors[0].name === "pattern") {
-            return getLocalizedString("core.QuestionAppName.validation.pattern");
+            return getLocalizedString("core.QuestionAppName.validation.pattern"); // TODO: update string
```

Comment:
@MuyangAmigo Hi Junjie, we need to update the value of this string. The current value is "Application name must start with a letter and can only contain letters and digits."

Reply:
Proposed message: `Application name must start with letters and contain at least two letters or digits. It can not contain specific ASCII characters.`